identity-composer-component.html

```diff
@@ -1,31 +1,137 @@
+<style>
```

Comment:
I guess this would be moved into a `less` file for this module once everyone signs off on the design.